frontend/vendor/confetti.js

```diff
@@ -388,7 +446,16 @@ if (typeof(window) !== 'undefined') {
         confettiRibbons[i].Update(duration);
         confettiRibbons[i].Draw(context);
       }
-      this.interval = rAF(this.update.bind(this));
+      framesToRegenerate--;
+      if (framesToRegenerate == 0) {
```

Comment:
Note that if `regenerateForSecs` was its default of `0`, this conditional will never evaluate to `true`, which is why the confetti will be eternal.  I'm not sure if I like this behavior because of how non-explicit it is, but whatevs, this code is already blargy.